### PR TITLE
[fix bug 1139040] Populate LANG and DIR context variables.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@jgmize r?

File: masterfirefoxos/base/context_processors.py [hunk 1]
```diff
@@ -1,7 +1,10 @@
 from django.conf import settings as django_settings
+from django.utils import translation
 
 
 def settings(request):
     return {
-        'settings': django_settings
+        'settings': django_settings,
+        'LANG': translation.get_language(),
+        'DIR': 'rtl' if translation.get_language_bidi() else 'ltr',
```
Comment (jgmize) [inline masterfirefoxos/base/context_processors.py]: NIT: this works, but wouldn't it make more sense in a separate context processor?

**Comment by jgmize** [comment]
r+wc

File: masterfirefoxos/base/context_processors.py [hunk 2]
```diff
@@ -1,7 +1,15 @@
 from django.conf import settings as django_settings
+from django.utils import translation
+
+
+def l18n(request):
```
Comment (jgmize) [inline masterfirefoxos/base/context_processors.py]: NIT: shouldn't this be either `i18n` or `l10n`?

**Pull request merged**